src/main/java/com/fabriccommunity/thehallow/registry/HallowedTags.java

```diff
@@ -24,11 +24,12 @@ private Items() {
 		private Blocks() {
 			// NO-OP
 		}
+    public static final Tag<Block> GATE_CIRCLE = registerBlock("gate_circle");
```

Comment:
No empty line before field & weird indenting?

Reply:
That seems to be from the upstream merge. Also is spacing.

Also we do no spacing from class starting to first field/constructor/method unless comments are present.